.github/actions/setup_node/action.yml

```diff
@@ -1,10 +1,14 @@
 # basic setup that applies to most jobs
 # checks out the repo and configures node
 name: setup_node
+inputs:
+  node-version:
+    description: node version used to configure environment with
+    default: lts/hydrogen
```

Comment:
can we use the node version numbers instead?

Reply:
they should both resolve to latest versions. I can update it to numbers